infra/binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/expression/ExpressionSegmentBinderTest.java

```diff
@@ -0,0 +1,124 @@
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.FunctionSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.InExpression;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.subquery.SubqueryExpressionSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.expr.subquery.SubquerySegment;
+import org.apache.shardingsphere.sql.parser.sql.common.segment.dml.item.ProjectionsSegment;
+import org.apache.shardingsphere.sql.parser.sql.common.value.identifier.IdentifierValue;
+import org.apache.shardingsphere.sql.parser.sql.dialect.statement.mysql.dml.MySQLSelectStatement;
+import org.apache.shardingsphere.test.fixture.database.MockedDatabaseType;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class ExpressionSegmentBinderTest {
+    
+    @Test
+    void testBindWithBinaryOperationExpression() {
+        BinaryOperationExpression binaryOperationExpression = new BinaryOperationExpression(0, 0, null, null, "+", "text");
+        SQLStatementBinderContext statementBinderContext = new SQLStatementBinderContext(new ShardingSphereMetaData(), "db", new MockedDatabaseType(), Collections.emptyList());
+        ExpressionSegment actual = ExpressionSegmentBinder.bind(binaryOperationExpression, SegmentType.INSERT_COLUMNS, statementBinderContext, Collections.emptyMap(), Collections.emptyMap());
+        assertTrue(actual instanceof BinaryOperationExpression);
+        assertEquals(binaryOperationExpression.getLeft(), ((BinaryOperationExpression) actual).getLeft());
+        assertEquals(binaryOperationExpression.getOperator(), ((BinaryOperationExpression) actual).getOperator());
+        assertEquals(binaryOperationExpression.getRight(), ((BinaryOperationExpression) actual).getRight());
+    }
+    
+    @Test
+    void testBindWithExistsSubqueryExpression() {
+        MySQLSelectStatement mySQLSelectStatement = new MySQLSelectStatement();
+        mySQLSelectStatement.setProjections(new ProjectionsSegment(0, 0));
+        ExistsSubqueryExpression existsSubqueryExpression = new ExistsSubqueryExpression(0, 0, new SubquerySegment(0, 0, mySQLSelectStatement, "test"));
+        SQLStatementBinderContext statementBinderContext = new SQLStatementBinderContext(new ShardingSphereMetaData(), "db", new MockedDatabaseType(), Collections.emptyList());
+        ExpressionSegment actual = ExpressionSegmentBinder.bind(existsSubqueryExpression, SegmentType.INSERT_COLUMNS, statementBinderContext, Collections.emptyMap(), Collections.emptyMap());
+        assertTrue(actual instanceof ExistsSubqueryExpression);
+        assertEquals(existsSubqueryExpression.getSubquery().getClass(), ((ExistsSubqueryExpression) actual).getSubquery().getClass());
```

Comment:
Please use assertThat instead of assertEquals.

Reply:
Updated assertThat